build/buildBuildImages.sh

```diff
@@ -48,46 +48,92 @@ fi
 
```

Comment:
We finally have a reason to call this file buildBuildImage*s*.sh

Reply:
LOL..yes